### ISSUE TITLE
Alabama: ReportStream to Baptist

### ISSUE THREAD

**Issue by sfradkin** [CDCgov/trusted-intermediary#830]
Depends on https://github.com/CDCgov/trusted-intermediary/issues/966 being completed.

**Comment by JohnNKing** [comment]
Confirmed network connectivity to Oracle RLN staging and production; authentication and results submission previously confirmed in staging / non-prod.

**Comment by JohnNKing** [comment]
Testing today indicated an issue with the jurisdiction filter and transport config in staging. I'm re-testing this now after @basiliskus updated both.

**Comment by JohnNKing** [comment]
Once we've updated our receiver config, we should be able to re-test with the sample OML messages that Bill emailed on 8/1 -- I'll forward that to the team now.

**Comment by JohnNKing** [comment]
Re-testing resulted in a message routing, but that was never delivered. IP whitelisting may well be the cause. See more details in https://www.slack.com/archives/C044P2CBEKS/p1723745872158119

**Comment by JohnNKing** [comment]
Next step: Follow-up with Oracle to clarify if whitelisting is still impacting staging.

**Comment by JohnNKing** [comment]
My understanding is this may actually be due to the receiver in ReportStream attempting to use OAuth2 for message delivery (when a static API key is needed). The team is working with ReportStream on a fix.

**Comment by JohnNKing** [comment]
1. The OAuth2 issue was fixed
2. Further configuration with receiver issues was identified and fixed
3. A credential issue was identified and fixed
4. Results were failing to be delivered due to connectivity timeouts to the RLN non-prod endpoint
5. We changed to a new endpoint and things appear to now be working!

We're now just awaiting confirmation from Oracle that they received the ORU successfully.
